src/main/resources/swagger/rawls.yaml

```diff
@@ -3326,7 +3368,15 @@ definitions:
         type: array
         items:
           type: string
-      groupEmails:
+      subGroupEmails:
```

Comment:
If you end up going with the four-field version of this then please add more words to the description explaining what's what and what's optional.